image/v3/modifying/text-fonts.md

```diff
@@ -97,7 +97,7 @@ Define the text color in one of the valid [color formats](/v3/introduction/forma
 
 ### Horizontal alignment
 
-> public FontInterface::align(string $align): FontInterface
+> public FontInterface::setAlignment(string $align): FontInterface
```

Comment:
Please revert. This is correct call of class Intervention\Image\Typography\FontFactory::class.